runtime/gateway.go

```diff
@@ -340,6 +340,15 @@ func (gateway *Gateway) setupMetrics(config *StaticConfig) (err error) {
 		"env":     env,
 		"service": service,
 	}
+	// Adds in any env variable variables specified in config
+	envVarsToTagInRootScope := []string{}
+	config.MustGetStruct("envVarsToTagInRootScope", &envVarsToTagInRootScope)
+	for _, envVarName := range envVarsToTagInRootScope {
+		envVarValue := os.Getenv(envVarName)
+		if envVarValue != "" {
```

Comment:
@ChuntaoLu I'm actually not sure we should have this `if` check.  I believe if the env var is in the config, we'd want to add it as a tag even if it doesn't exist -- if the tag doesn't exist in those cases, I think that's less consistent metrics behavior than if we tag it with an empty string.  Open to other opinions, though, LMK your thoughts.